src/main/java/org/hmhb/audit/DefaultAuditHelper.java

```diff
@@ -0,0 +1,35 @@
+package org.hmhb.audit;
+
+import java.util.Date;
+
+import com.codahale.metrics.annotation.Timed;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DefaultAuditHelper implements AuditHelper {
```

Comment:
I created this for an easier time in my unit tests.  I will create a ticket to do something more meaningful than hardcoding the ip and user.